config-controller/api/v1alpha1/policy_types.go

```diff
@@ -18,11 +18,200 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/stackrox/rox/generated/storage"
 )
 
 // PolicySpec defines the desired state of Policy
 type PolicySpec struct {
-	Policy string `json:"policy,omitempty"`
+	Name        string   `json:"name,omitempty"`
```

Comment:
We'll need the comment in the policy.proto since anyone adding a field in there must add one to this struct (oh how I wish that DeepCopy issue could be worked around and we could have used the generated proto struct!)

Reply:
[a57f884](https://github.com/stackrox/stackrox/pull/12647/commits/a57f88401ac1563a5862cdc6cd37319cb3de609c)